website/content/ja/_index.md

```diff
@@ -9,45 +9,41 @@ description: "TEST"
 <div class="row mt-5 mb-3">
     <div class="col-lg-6">
         <div class="lead">
-        TAG Runtime objective is to enable widespread and successful execution of the full spectrum of workload types, including both general latency-sensitive and batch, as well as more specialized categories listed as in scope, in cloud-native environments.
+        TAG Runtimeは、一般的なレイテンシー重視のワークロードやバッチ、クラウドネイティブ環境のスコープ内とされるより専門的なカテゴリを含むあらゆる種類のワークロードを幅広くかつ正常に実行できるようにすることです。
         </div>
     </div>
     <div class="col-lg-6">
         <img src="/images/tag-runtime_icon-color.svg" alt="Tag App Delivery logo" style="max-width: 300px;">
     </div>
 </div>
 
+このTAGはクラウドアプリケーションユーザーや開発者向けにガイダンスを提供し、フィードバックを収集します。
+また、TAGの技術領域でCNCFプロジェクトへのガイダンスと調整を行ないます。
 
-The TAG produces guidance for and gathers feedback from cloud app users and
-developers and provides guidance and coordination to CNCF projects in the TAG's
-technical domains.
-
-- [TAG Charter](https://github.com/cncf/toc/blob/main/tags/runtime-charter.md)
-- [Community events](https://community.cncf.io/tag-runtime/)
-- Slack channel: [#tag-runtime](https://cloud-native.slack.com/messages/CPBE97SMU)
-    - [Invite yourself to the CNCF Slack](https://slack.cncf.io/)
-- [Mailing list](https://lists.cncf.io/g/cncf-runtime/topics)
+- [TAG チャーター](https://github.com/cncf/toc/blob/main/tags/runtime-charter.md)
+- [コミュニティイベント](https://community.cncf.io/tag-runtime/)
+- Slackチャンネル: [#tag-runtime](https://cloud-native.slack.com/messages/CPBE97SMU)
+    - [CNCF Slackにご自身を招待する](https://slack.cncf.io/)
+- [メーリングリスト](https://lists.cncf.io/g/cncf-runtime/topics)
 
 <p class="mt-5"><img src="/images/man-using-laptop.jpg" alt="Man working on computer"></p>
 
 
-## Meetings
+## ミーティング
 
-1st and 3rd Thursday of each month at 8:00 PT ([convert to your local
-time](https://dateful.com/convert/pacific-time-pt?t=08)).
+毎月の第1,第3木曜日8:00 PT ([居住地の時刻に変換](https://dateful.com/convert/pacific-time-pt?t=08)).
 
-Meetings are listed on the [main CNCF calendar](https://www.cncf.io/calendar/)
-as well as the [CNCF Community Calendar](https://community.cncf.io/tag-runtime/).
+ミーティングは[CNCFカレンダー](https://www.cncf.io/calendar/)と[CNCFコミュニティカレンダー](https://community.cncf.io/tag-runtime/)に掲載されます。
 
-* [Agenda and Notes](https://bit.ly/cncf-tag-runtime-meeting-notes)
-* [Zoom Meeting](https://zoom.us/j/9890721462?pwd=N2xyRkZaN2JWZkNmS3EzbE1HVnhEQT09) (Passcode: 77777)
-* [Recordings of previous meetings](https://www.youtube.com/@cncftagruntime173/videos)
+* [アジェンダと議事録](https://bit.ly/cncf-tag-runtime-meeting-notes)
+* [Zoomミーティング](https://zoom.us/j/9890721462?pwd=N2xyRkZaN2JWZkNmS3EzbE1HVnhEQT09) (パスコード: 77777)
+* [過去のミーティングの録画](https://www.youtube.com/@cncftagruntime173/videos)
 
-## Leads
+## リーダー
```

Comment:
```suggestion
## リード
```

Which do you think is better, `リーダー` or `リード`? Since `TL (Tech Lead)` is translated as `テックリード`, so `リード` would be better.

Reply:
Agreed. It would make more sense to translate to `リード`